TypeCobol/AntlrGrammar/CobolCodeElements.g4

```diff
@@ -8305,6 +8306,12 @@ lockTableStatement: SQL_LOCK SQL_TABLE tableOrViewOrCorrelationName (SQL_PARTITI
 share: ({ string.Equals(CurrentToken.Text, "SHARE", System.StringComparison.OrdinalIgnoreCase) }? KeywordSHARE=UserDefinedWord);
 exclusive: ({ string.Equals(CurrentToken.Text, "EXCLUSIVE", System.StringComparison.OrdinalIgnoreCase) }? KeywordEXCLUSIVE=UserDefinedWord);
 sql_mode: ({ string.Equals(CurrentToken.Text, "MODE", System.StringComparison.OrdinalIgnoreCase) }? KeywordMODE=UserDefinedWord);
+sqlRetain: ({ string.Equals(CurrentToken.Text, "RETAIN", System.StringComparison.OrdinalIgnoreCase) }? KeywordRETAIN=UserDefinedWord);
+sqlCursors: ({ string.Equals(CurrentToken.Text, "CURSORS", System.StringComparison.OrdinalIgnoreCase) }? KeywordCURSORS=UserDefinedWord);
+sqlLocks: ({ string.Equals(CurrentToken.Text, "LOCKS", System.StringComparison.OrdinalIgnoreCase) }? KeywordLOCKS=UserDefinedWord);
+onRollbackRetain: SQL_ON SQL_ROLLBACK sqlRetain;
+savepointStatement : SQL_SAVEPOINT (savepoint_name=UserDefinedWord) SQL_UNIQUE? onRollbackRetain ((sqlCursors (onRollbackRetain sqlLocks)?) | (sqlLocks onRollbackRetain sqlCursors));
```

Comment:
It doesn't works like that ?
```suggestion
savepointStatement : SQL_SAVEPOINT (savepoint_name=UserDefinedWord) SQL_UNIQUE? onRollbackRetain sqlCursors (onRollbackRetain sqlLocks)?;
```

Reply:
It doesn't work because "ON ROLLBACK RETAIN CURSORS" and "ON ROLLBACK RETAIN LOCKS" can be specified in either order so we can have sqlLocks before sqlCursors .

Reply:
After test, it doesn't seems to be allowed even if the documentation say otherwise.
The order `ON ROLLBACK RETAIN CURSORS` then `ON ROLLBACK RETAIN LOCKS` is mandatory.